apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html

```diff
@@ -137,26 +137,26 @@ <h3 class="modal-title pull-left">Rejection Reason</h3>
                 <span>by {{selectedItem?.createdBy?.name}}</span>
             </p>
             <div class="row mb-1">
-                <div class="col-xs-4">
+                <div class="col-xs-4 col-sm-4">
                     <p class="font-14">Expenses:</p>
                 </div>
                 <div class="col-xs-8">
```

Comment:
please verify once:- 
isn't it we required **col-sm-8** class inside this div ?

Reply:
Done